composer.json

```diff
@@ -17,11 +17,12 @@
         "illuminate/routing": "5.5.x|5.6.x|5.7.x|5.8.x",
         "illuminate/support": "5.5.x|5.6.x|5.7.x|5.8.x",
         "illuminate/view": "5.5.x|5.6.x|5.7.x|5.8.x",
-        "psr/log": "^1.1",
         "ramsey/uuid": "^3.0",
+        "psr/log": "^1.0",
         "scoutapp/scout-apm-php": "0.2.0"
     },
     "require-dev": {
+        "psr/log": "^1.1",
```

Comment:
We don't need this in the `laravel` package, can be removed from `require-dev` for us.